src/components/range-slider/range-slider.tsx

```diff
@@ -34,9 +34,7 @@ const RangeSlider = ({
 }: Props) => {
   const theme = useTheme<ITheme>();
 
-  const handleStyle = useMemo(buildHandleStyle(theme.colors.primary), [
-    theme.colors.primary,
-  ]);
+  const handleStyle = buildHandleStyle(theme.colors.primary);
```

Comment:
@snqb actually this is not a heavy task so useMemo doesn't improve the performance but the opposite

Reply:
yeah, you're right